classes/module/Module.php

```diff
@@ -185,6 +185,9 @@
     /** @var \Symfony\Component\DependencyInjection\ContainerInterface */
     private $container;
 
+    /** @var array|null used to cache module ids */
+    private static $id2name = null;
```

Comment:
not a big fan of the naming, if you don't have choice but to keep a static (even though a class static is less ugly) could you use the opportunity to rename it? `moduleNames` or `hashModuleNames`? or maybe `cachedModuleNames`?

Reply:
We can refactoring this later :/

Reply:
we never will ^^

Reply:
Le Fu!! But done for your beautiful eyes!

Reply:
besides it caches the names of the module not their ids.. the value is the name